code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1429,7 +1429,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			H.domutcheck()
 
 	if(radiation > RAD_MOB_HAIRLOSS)
-		if(prob(15) && !(H.hair_style == "Bald") && (HAIR in species_traits))
+		if(prob(15) && !(H.hair_style == "Bald") && (HAIR in species_traits) && !HAS_TRAIT(H, TRAIT_TOXINLOVER))
```

Comment:
Toxinlover just prevents the tox damage from hurting oozelings - radiation still acts on them and they still gain rads. They can still mutate, vomit, etc.

I'm really of the opinion that they should still take damage from rads if they don't, since the toxin damage from rads isn't actually supposed to be 'toxin' but a result of their DNA getting mangled to shit - essentially, cancer.

If we're going the direction that "oozeling hair is just jelly", then we should specifically just check `isoozeling()` rather than relying on the trait.

Reply:
I agree with Kat, this should be a mob trait that indicates the species has hair which is part of their body instead of an actual hair thing. I don't think it should be a specific isoozeling check.

Reply:
moved it to a separate trait, and made oozelings unaffected by alopecia as well